app/controllers/api/articles_controller.rb

```diff
@@ -30,6 +47,35 @@ def create
 
   private
 
+  def check_index_params
```

Comment:
Maybe this could be done in the model for the article instead, to keep the controller slimmer....?

Reply:
Hm, can we do that on an index request? Isn't that just for validating columns when created?